functional_tests/sb3-integration.py

```diff
@@ -3,23 +3,16 @@
 
 ---
 id: 0.0.4
-check-ext-wandb:
-  run:
-    - exit: 0
-      config:
-        policy_type: {desc: null, value: "MlpPolicy"}
-        total_timesteps: {desc: null, value: 200}
-        policy_class: {desc: null, value: "<class 'stable_baselines3.common.policies.ActorCriticPolicy'>"}
-        action_space: {desc: null, value: "Discrete(2)"}
-        batch_size: {desc: null, value: 64}
-        n_epochs: {desc: null, value: 10}
-
```

Comment:
nice. what was the fix here?

Reply:
When i converted to "assert" checks and removed the ignore_ parameters things just worked:

This is the yea changes: wandb/yea-wandb#6

Reply:
nice.